app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_data: list) -> list:
+    person_list = []
+    for person_dict in people_data:
+        person = Person(person_dict["name"], person_dict["age"])
+        person_list.append(person)
```

Comment:
This violates checklist item https://github.com/mate-academy/py-person-class/pull/3: "Use a list comprehension to build the list of Person instances from the input dictionaries." You are currently using a for loop to create and append Person instances. Please replace this with a list comprehension as required.